### ISSUE TITLE
Error inside LightTable

### ISSUE THREAD

**Issue by oeb25** [rundis/elm-light#3]
I have all of the requirements installed and elm at v0.15.1 installed with the Mac installer.

When I run any elm releated commands in LightTable I get the same error.

My LightTable is freashly installed on version 0.7.2, running OSX 10.11.1.

The error message is the following when running elm lint-

```
We couldn't connect.

Looks like there was an issue trying to connect to the project. Here's what we got:
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn elm-reactor ENOENT
    at exports._errnoException (util.js:837:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:437:9)
    at process._tickCallback (node.js:351:17)
```

My folder structure is

```
├── Main.elm
├── elm-package.json
├── elm.js
└── elmclient.log
```

And the content of my `elm-package.json` is

```
{
    "version": "1.0.0",
    "summary": "helpful summary of your project, less than 80 characters",
    "repository": "https://github.com/user/project.git",
    "license": "BSD3",
    "source-directories": [
        "."
    ],
    "exposed-modules": [],
    "dependencies": {
        "elm-lang/core": "2.1.0 <= v < 3.0.0",
        "evancz/elm-html": "4.0.1 <= v < 5.0.0",
        "evancz/start-app": "2.0.1 <= v < 3.0.0"
    },
    "elm-version": "0.16.0 <= v < 0.17.0"
}
```

**Comment by oeb25** [comment]
Just realized **_LIghtTable 0.8-alpha or higher is required**_. Sorry about that :+1:

**Comment by ai-pavel** [comment]
Experiencing same issue with 0.8.0 alpha on Linux. Have all necessary binaries in my `$PATH`

**Comment by rundis** [comment]
https://nodejs.org/api/errors.html#errors_enoent_no_such_file_or_directory
I'm pretty sure this error is an indication that elm-reactor cannot be found (from the path available to Light Table).

I just tried on a different mac. Tried first without installing elm and got the exact same error, then installed elm and it worked just fine. Haven't tried linux yet, but will give it a test run when I get a chance.
